modules/platform/nuxeo-platform-task/nuxeo-platform-task-api/src/main/java/org/nuxeo/ecm/platform/task/TaskService.java

```diff
@@ -254,14 +255,31 @@ String endTask(CoreSession coreSession, NuxeoPrincipal principal, Task task, Str
      *
      * @since 5.7.3
      */
-    void reassignTask(CoreSession session, String taskId, List<String> actors, String comment);
+    default void reassignTask(CoreSession session, String taskId, List<String> actors, String comment) {
+        reassignTask(session,taskId,actors,comment,new HashMap<>());
```

Comment:
Format

Reply:
Also, could you change the `new HashMap` by a `Map.of()` (or a `Collections.emptyMap()` if you want to have the same code than the backport)?